tagpr.go

```diff
@@ -606,11 +606,12 @@ func (tp *tagpr) generatenNextLabels(prIssues []*github.Issue) []string {
 	return nextLabels
 }
 
-func buildChunkSearchIssuesQuery(queryBase string, shasStr string) (chunkQueries []string) {
-	query := queryBase
+func buildChunkSearchIssuesQuery(qualifiers string, shasStr string) (chunkQueries []string) {
+	// array of SHAs
+	keywords := make([]string, 0, 25)
```

Comment:
To clarify that the character limit applies only to the part of the string that makes up the query, excluding the qualifier, the variable name has been changed.